pkg/apis/datadoghq/v1alpha1/const.go

```diff
@@ -66,27 +67,45 @@ const (
 	KubernetesEnvvarName = "KUBERNETES"
 
 	// Datadog volume names and mount paths
-	ConfdVolumeName           = "confd"
-	ConfdVolumePath           = "/conf.d"
-	ConfigVolumeName          = "config"
-	ConfigVolumePath          = "/etc/datadog-agent"
-	ProcVolumeName            = "procdir"
-	ProcVolumePath            = "/host/proc"
-	ProcVolumeReadOnly        = true
-	CgroupsVolumeName         = "cgroups"
-	CgroupsVolumePath         = "/host/sys/fs/cgroup"
-	CgroupsVolumeReadOnly     = true
-	CriSockerVolumeName       = "runtimesocket"
-	CriSockerVolumeReadOnly   = true
-	DogstatsdSockerVolumeName = "dsdsocket"
-	DogstatsdSockerVolumePath = "/var/run/datadog"
-	PointerVolumeName         = "pointerdir"
-	PointerVolumePath         = "/opt/datadog-agent/run"
-	LogPodVolumeName          = "logpodpath"
-	LogPodVolumePath          = "/var/log/pods"
-	LogPodVolumeReadOnly      = true
-	LogContainerVolumeName    = "logcontainerpath"
-	LogContainerolumeReadOnly = true
+	ConfdVolumeName                    = "confd"
+	ConfdVolumePath                    = "/conf.d"
+	ConfigVolumeName                   = "config"
+	ConfigVolumePath                   = "/etc/datadog-agent"
+	ProcVolumeName                     = "procdir"
+	ProcVolumePath                     = "/host/proc"
+	ProcVolumeReadOnly                 = true
+	PasswdVolumeName                   = "passwd"
+	PasswdVolumePath                   = "/etc/passwd"
+	CgroupsVolumeName                  = "cgroups"
+	CgroupsVolumePath                  = "/host/sys/fs/cgroup"
+	CgroupsVolumeReadOnly              = true
+	SystemProbeSocketVolumeName        = "sysprobe-socket-dir"
+	SystemProbeSocketVolumePath        = "/opt/datadog-agent/run"
+	CriSockerVolumeName                = "runtimesocket"
+	CriSockerVolumeReadOnly            = true
+	DogstatsdSockerVolumeName          = "dsdsocket"
+	DogstatsdSockerVolumePath          = "/var/run/datadog"
+	PointerVolumeName                  = "pointerdir"
+	PointerVolumePath                  = "/opt/datadog-agent/run"
+	LogPodVolumeName                   = "logpodpath"
+	LogPodVolumePath                   = "/var/log/pods"
+	LogPodVolumeReadOnly               = true
+	LogContainerVolumeName             = "logcontainerpath"
+	LogContainerolumeReadOnly          = true
```

Comment:
```suggestion
	LogContainerVolumeReadOnly          = true
```
an old typo in the code